tools/dhtnode.cpp

```diff
@@ -253,6 +257,7 @@ void cmd_loop(std::shared_ptr<DhtRunner>& dht, dht_params& params
         dht::InfoHash id;
 
         if (op == "il" or op == "ii") {
+#ifdef OPENDHT_INDEXATION
```

Comment:
Not mendatory, but I will prefer the if condition (op == "ii" and il) in the #ifdef to avoid any build warnings (useless if).

Reply:
done